src/main/kotlin/com/team4099/robot2023/subsystems/elevator/Elevator.kt

```diff
@@ -0,0 +1,356 @@
+package com.team4099.robot2023.subsystems.elevator
+
+import com.team4099.lib.hal.Clock
+import com.team4099.lib.logging.LoggedTunableNumber
+import com.team4099.lib.logging.LoggedTunableValue
+import com.team4099.robot2023.config.constants.Constants
+import com.team4099.robot2023.config.constants.ElevatorConstants
+import edu.wpi.first.units.Voltage
+import edu.wpi.first.wpilibj.RobotBase
+import edu.wpi.first.wpilibj2.command.Command
+import edu.wpi.first.wpilibj2.command.Commands
+import edu.wpi.first.wpilibj2.command.Commands.run
+import edu.wpi.first.wpilibj2.command.Commands.runOnce
+import com.team4099.robot2023.subsystems.superstructure.Request.ElevatorRequest as ElevatorRequest
+import org.team4099.lib.controller.ElevatorFeedforward
+import org.team4099.lib.controller.TrapezoidProfile
+import org.team4099.lib.units.base.Meter
+import org.team4099.lib.units.base.inches
+import org.team4099.lib.units.derived.*
+import org.team4099.lib.units.perSecond
+import org.team4099.lib.units.base.inInches
+import org.littletonrobotics.junction.Logger
+import org.team4099.lib.units.base.inSeconds
+import org.team4099.lib.units.inInchesPerSecond
+import kotlin.time.Duration.Companion.seconds
+
+class Elevator(val io: ElevatorIO) {
+    val inputs = ElevatorIO.ElevatorInputs()
+    private var elevatorFeedforward : ElevatorFeedforward =
```

Comment:
can we make Tunable number units for FF as well. going to be useful when you need to tune ks, adjust kV, etc. better have it and not need it than redeploy for one number change

Reply:
what should the units be for the kA logged tunable value